othello/apps/games/tasks.py

```diff
@@ -117,11 +118,12 @@ def run_game(game_id):
                 exception = e
 
             for log in running_turn:
+                print(log)
                 game.logs.create(
                     player=current_player.value, message=log,
                 )
                 send_through_game_channel(game, "game.log")
-                sleep(0.05)
+                sleep(1)
```

Comment:
This cause a race condition bug where the previous player's logs where ignored and the current player's logs were sent multiple times. 

For example, I did 2021abagali vs Yourself. My AI prints out two logs and Yourself prints out one log. At `sleep(0.05)` my logs were ignored and the Yourself log printed out three times. Three `game.log` events where triggered but the code was so fast such that the `GameLog.objects.latest()` returned Yourself's log instead of my AI's log.

Possible fix: Is there a way to send a parameter with the event? i.e 
```
log = game.logs.create( ... )
send through_game_channel(game, "game.log", log.id)
```
and then in `consumers.py`, I would retrieve that specific `GameLog` by id instead of doing `GameLog.objects.latest()` which is heavily time dependent.

Reply:
implemented this, works